example/README.md

```diff
@@ -0,0 +1,16 @@
+# clevertap_signedcall_flutter_example
+
+Demonstrates how to use the clevertap_signedcall_flutter plugin.
+
+## Getting Started
```

Comment:
@shivamsharma2710 remove/update the Getting Started section

Reply:
done.